bitnami/nats/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "nats.fullname" . }}
+  labels:
+    app: "{{ template "nats.name" . }}"
+    chart: "{{ template "nats.chart" . }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "nats.name" . }}"
+      release: {{ .Release.Name | quote }}
+{{ toYaml .Values.podDisruptionBudget | indent 2 }}
```

Comment:
This works well for sure, but I think the most common approach followed by the majority of Charts is to split this into `minAvailable` and `maxUnavailable`. You can see an example in the [consul](https://github.com/bitnami/charts/blob/1f6bb260ebe2f43808939a60a307a24a124c1b24/bitnami/consul/templates/pod-dist-budget.yaml#L9) chart.

https://github.com/bitnami/charts/blob/1f6bb260ebe2f43808939a60a307a24a124c1b24/bitnami/consul/templates/pod-dist-budget.yaml#L9-L14

Could you please change this to follow an standardized approach?